resolwe_bio/docker_images/latex/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM resolwe/base:ubuntu-16.04
+
+RUN apt-get update && apt-get upgrade -y && apt-get install -y \
+    wget \
+    texlive-latex-base \
+    texlive-latex-recommended \
+    texlive-latex-extra
+
+# Cleanup.
+RUN apt-get clean && rm -rf /var/lib/apt/lists/*
```

Comment:
Combine two `RUN` commands.

Each command creates a layer of docker image. So if you clean the image in separate command, all unnecessary things will be presented in the intermediate layer and you won't save anything on the space.